components/suggestion/suggestion_list.jsx

```diff
@@ -65,6 +65,10 @@ export default class SuggestionList extends React.PureComponent {
             const contentBottomPadding = parseInt(content.css('padding-top'), 10);
 
             const item = $(ReactDOM.findDOMNode(this.refs[term]));
+            if (!item[0]) {
```

Comment:
Minor nit, but could this be `item.length === 0`? It might just be a bit clearer